memoize/__init__.py

```diff
@@ -0,0 +1 @@
+from .memoized import Memoized
```

Comment:
What's this?

Reply:
Enables [shorter imports when using the package](https://stackoverflow.com/a/35733111/51197). The caller uses `from memoized import Memoized` instead of `from memoize.memoized import Memoized` because it's in the top-level of the package, inside the `__init__.py` file.

See [the change in our example file](https://github.com/IttayEyal/memoization/pull/34/files#diff-6a6e57c6fff6087e2cac547e260c49d313fd40975812f8f9b936c5df06233b15L1).